### PR TITLE
[WIP]-DOCTEAM-1552:creates the public cloud chapter and adds to the book

### PR DESCRIPTION
**Pull request by Amrita42**
### PR creator: Description

Describe the overall goals of this pull request.

### PR creator: Are there any relevant issues/feature requests?

[DOCTEAM-1552](https://jira.suse.com/browse/DOCTEAM-1552)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP6/openSUSE Leap 15.6
  - [x] SLE 15 SP5/openSUSE Leap 15.5
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2

- SLE 12
  - [ ] SLE 12 SP5

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by Amrita42** [comment]
[book-rmt_en.pdf](https://github.com/user-attachments/files/17303630/book-rmt_en.pdf)

**Comment by Amrita42** [comment]
[book-rmt_en.pdf](https://github.com/user-attachments/files/17376438/book-rmt_en.pdf)

**Comment by Amrita42** [comment]
@dariavladykina please review .

File: xml/rmt_public_cloud.xml [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  The security group must be configured to allow inbound access to the &rmt; server from your clients.
+  For GCP, the firewall rules must be configured to allow inbound access to the &rmt; server from your clients.
+   </para>
+         </listitem>
+  </varlistentry>
+</variablelist>
+</sect1>
+<sect1>
+  <title>More information</title>
+  <variablelist>
```
Comment (dariavladykina) [inline xml/rmt_public_cloud.xml:149]: I'd suggest using an `itemizedlist` here, as it is generally a convention to first mention the item of interest and then provide the link to it.

```
<itemizedlist>
      <listitem>
        <para>
          Find organization credentials to create a local mirror of the &sle; repositories
          at <link xlink:href="https://documentation.suse.com/sles/15-SP6/html/SLES-all/cha-rmt-mirroring.html#sec-rmt-mirroring-credentials"/>.
        </para>
      </listitem>
      <listitem>
        <para>
          Register and activate &sles; with the <link xlink:href="https://documentation.suse.com/sles/15-SP6/html/SLES-all/cha-register-sle.html#sec-register-sle-installation/">SUSE Customer Center</link>.
        </para>
      </listitem>
    </itemizedlist>
```

File: xml/rmt_public_cloud.xml [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+ xml:id="rmt-public-cloud">
+  <title>Deploying a &sls; instance to support &rmt; in the public cloud</title>
+  <info>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+  </info>
+  <para>
+You can deploy a &sls; instance to support &rmt; in Amazon Web Services(AWS), Google Cloud Platform(GCP), and Microsoft Azure.
+The Repository Mirroring Tool (RMT) is included in &sls; starting with version 15. </para>
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply xml/rmt_public_cloud.xml:19]: Yes , a good suggestion and implemented , thanks

File: xml/rmt_public_cloud.xml (hunk 1 above)
Comment (Amrita42) [reply xml/rmt_public_cloud.xml:151]: Done

File: xml/rmt_public_cloud.xml [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
+ xml:id="rmt-public-cloud">
+  <title>Deploying a &sls; instance to support &rmt; in the public cloud</title>
+  <info>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+  </info>
+  <para>
+You can deploy a &sls; instance to support &rmt; in Amazon Web Services (AWS), Google Cloud Platform (GCP), and Microsoft Azure.
```
Comment (Amrita42) [inline xml/rmt_public_cloud.xml:18]: xxxxxx

**Pull request merged**